problems/countThrees.js

```diff
@@ -13,8 +13,15 @@
 * returns: 0
 */
 
-function countThrees() {
-
+const countThrees = (n) => {
+    let count = 0;
+    let str = n.toString();
+  for(let i = 0; i <= str.length; i++){
+      if("3".includes(str[i])){
+          count += 1;
+      }
+      }
```

Comment:
This works, we could have also said, `if(str[i] === "3")`
Also make sure to always have good formatted code